### PR TITLE
Update to use llvm 11 and monorepo layout

### PR DESCRIPTION
**Pull request by AlexeySachkov**
Also refactored README

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,10 +26,10 @@ addons:
         key_url: 'https://apt.llvm.org/llvm-snapshot.gpg.key'
       - ubuntu-toolchain-r-test
     packages:
-      - llvm-10-tools
-      - llvm-10-dev
-      - libclang-10-dev
-      - lldb-10
+      - llvm-11-tools
```
Comment (AlexeySotkin) [inline .travis.yml]: Can we reuse approach used in the translator - https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/33d49464beee75e02529658ea6682eec15a1cf93/.travis.yml#L29-L33

File: README.md [hunk 2]
```diff
@@ -16,27 +16,24 @@ Before the build all dependencies must be downloaded and laid out as follows:
...
 ```
 <workspace>
-`-- llvm
-    |-- tools
-    |   `-- clang
-    `-- projects
-        |-- llvm-spirv
-        `-- opencl-clang
+|-- llvm
+|   `-- projects
```
Comment (AlexeySotkin) [inline README.md]: I think it is better to place llvm-spirv and opencl-clang on the top level as it is done in https://github.com/intel/llvm/

File: .travis.yml (hunk 1 above)
Comment (AlexeySachkov) [reply .travis.yml]: Done

File: README.md (hunk 2 above)
Comment (AlexeySachkov) [reply README.md]: Done

**Pull request merged**